Triangulator/Source Files/Reader.cpp

```diff
@@ -2,11 +2,13 @@
 #include <iostream>
 #include <string>
 #include "Reader.h"
+
 using namespace std;
-vector<double> points;
-vector<Point> Reader::read() {
-    vector<Point> pstack;
-    unordered_map<double, int> umap;
+vector<double> unique_points;
+
+vector<Point> Reader::read_stl_file() {
+    vector<Point> point_list;
+    unordered_map<double, int> unique_value_map;
     ifstream myfile("cube-ascii.stl");

```

Comment:
Filename is hardcoded, if you have to use different file at different times, you will have to come here change the name each time. File name should be the parameter entered in main function.